src/pages/docs/liveobjects/rest-api-usage.mdx

```diff
@@ -884,15 +533,15 @@ For `COUNTER_CREATE`, the `data` field should be a JSON object that contains the
 ```
 </Code>
 
-When you create a new object it is important that the new object is assigned to the object tree so that it is [reachable](/docs/liveobjects/concepts/objects#reachability) from the root object.
+When you create a new object it is important that the new object is assigned to the object tree so that it is [reachable](/docs/liveobjects/concepts/object#reachability) from the root object.
```

Comment:
When you create a new object it is important that the new object instance is assigned to the channel object so that it is [reachable](/docs/liveobjects/concepts/object#reachability).